AspNetCoreHero.Boilerplate.Application/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -14,6 +15,8 @@ public static void AddApplicationLayer(this IServiceCollection services)
             services.AddValidatorsFromAssembly(Assembly.GetExecutingAssembly());
             services.AddMediatR(Assembly.GetExecutingAssembly());
             //services.AddTransient(typeof(IPipelineBehavior<,>), typeof(ValidationBehavior<,>));
+            services.AddTransient(typeof(IPipelineBehavior<,>), typeof(LoggingBehaviour<,>));
+
```

Comment:
not needed line

Reply:
Additionaly I'm not sure that is proper place to do it. There could be more suitable place.
 For an example create method like below:

```C#
public static void AddMediatr(this IServiceCollcetion services){
   services.AddMediatr(Assembly...
   services.AddTransient(typeof(IPipelineBehavior....
}
```
and use it in `AddApplicationLayer` method instead of line 16. (services.AddMediatr...)